crates/store/src/db/mod.rs

```diff
@@ -266,29 +321,21 @@ impl Db {
             "Connected to the database"
         );
 
-        let conn = pool.get().await.map_err(DatabaseError::MissingDbConnection)?;
+        let me = Db { pool };
+        me.framed_without_transaction("migrations", apply_migrations).await?;
 
-        conn.interact(apply_migrations).await.map_err(|err| {
-            DatabaseError::InteractError(format!("Migration task failed: {err}"))
-        })??;
-
-        Ok(Db { pool })
+        // TODO rationalize magic numbers, and make them
```

Comment:
Dangling todo or is this future work?

Reply:
Looking at the code this is very out of place, deleted